cmd/kubebuilder/initproject/project.go

```diff
@@ -0,0 +1,142 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package initproject
+
+import (
+	"fmt"
+	"log"
+	"os"
+	"os/exec"
+	"strings"
+
+	"github.com/kubernetes-sigs/kubebuilder/cmd/kubebuilder/util"
+	"github.com/spf13/cobra"
+	flag "github.com/spf13/pflag"
+	"sigs.k8s.io/controller-tools/pkg/scaffold"
+	"sigs.k8s.io/controller-tools/pkg/scaffold/input"
+	"sigs.k8s.io/controller-tools/pkg/scaffold/manager"
+	"sigs.k8s.io/controller-tools/pkg/scaffold/project"
+)
+
+var prj *project.Project
+var bp *project.Boilerplate
+var gopkg *project.GopkgToml
+var mrg *manager.Cmd
+var dkr *manager.Dockerfile
+
+// ProjectCmd represents the project command
+var ProjectCmd = &cobra.Command{
+	Use:   "init",
+	Short: "Scaffold a new project.",
+	Long: `Scaffold a project.
+
+Writes the following files:
+- a boilerplate license file
+- a PROJECT file with the domain and repo
+- a Makefile to build the project
+- a Gopkg.toml with project dependencies
+- a cmd/manager/main.go to run
+
+project will prompt the user to run 'dep ensure' after writing the project files.
+`,
+	Example: `# Scaffold a project using the apache2 license with "The Kubernetes authors" as owners
+kubebuilder init --domain k8s.io --license apache2 --owner "The Kubernetes authors"
+`,
+	Run: RunInit,
+}
+
+func RunInit(cmd *cobra.Command, args []string) {
+	if util.IsNewVersion() {
+		fmt.Println("kubebuilder init can't be run again on an initialized project")
```

Comment:
May be re-word it: Failed to initialize project because project is already initialized ?

Reply:
done